apps/src/javalab/JavalabEditor.jsx

```diff
@@ -30,62 +31,27 @@ import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import _ from 'lodash';
 import msg from '@cdo/locale';
 import javalabMsg from '@cdo/javalab/locale';
+import HeightResizer from '@cdo/apps/templates/instructions/HeightResizer';
 import {CompileStatus} from './constants';
 import {makeEnum} from '@cdo/apps/utils';
 
-const style = {
-  editor: {
-    width: '100%',
-    height: 400,
-    backgroundColor: color.white
-  },
-  darkBackground: {
-    backgroundColor: color.dark_slate_gray
-  },
-  fileMenuToggleButton: {
-    margin: '0, 0, 0, 4px',
-    padding: 0,
-    height: 20,
-    width: 13,
-    backgroundColor: 'transparent',
-    border: 'none',
-    ':hover': {
-      cursor: 'pointer',
-      boxShadow: 'none'
-    }
-  },
-  darkFileMenuToggleButton: {
-    color: color.white
-  },
-  fileTypeIcon: {
-    margin: 5
-  },
-  tabs: {
-    backgroundColor: color.background_gray,
-    marginBottom: 0,
-    display: 'flex',
-    alignItems: 'center'
-  },
-  backpackSection: {
-    textAlign: 'left',
-    display: 'inline-block',
-    float: 'left',
-    overflow: 'visible',
-    marginLeft: 3
-  }
-};
-
+const MIN_HEIGHT = 100;
+const MAX_HEIGHT = 500;
```

Comment:
It may make more sense to set Max_height instead as a percentage of the screen. i.e. for very tall or very short screens this would feel a little odd.

Reply:
Okay if this is part of a future iteration? This PR doesn't take into account the size of the console or the height of the run/settings buttons yet either. I've created a ticket for it here: https://codedotorg.atlassian.net/browse/CSA-440

Reply:
Sounds good. Is that a follow-up change you're planning to do or should that get picked up by someone else?

Reply:
It's currently assigned to me- am happy to take it on.

Reply:
👍 I broke out one task in particular - resizing the console when the resizer is used. I'll see who has space in their sprint to pick it up: https://codedotorg.atlassian.net/browse/CSA-446